silkrpc/core/rawdb/chain.cpp

```diff
@@ -389,4 +389,32 @@ boost::asio::awaitable<Transactions> read_noncanonical_transactions(const Databa
     co_return txns;
 }
 
+boost::asio::awaitable<intx::uint256> read_total_issued(const core::rawdb::DatabaseReader& reader, uint64_t block_number) {
+    const auto block_key = silkworm::db::block_key(block_number);
+    const auto kv_pair = co_await reader.get(db::table::kIssuance, block_key);
+    intx::uint256 total_issued = 0;
+    if (kv_pair.value != silkworm::ByteView{}) {
```

Comment:
Replace `kv_pair.value != silkworm::ByteView{}` with `kv_pair.value.empty()` because the former is tricky. What happens under the hood is:
- `kv_pair.value` has type `silkworm::Bytes` but is convertible to `silkworm::ByteView`, so a temporary `silkworm::ByteView` instance wrapped around it is created
- then, such instance is compared with a temporary empty `silkworm::ByteView` instance

Using `silkworm::Bytes::empty` method is clearer and avoids 2 temporary object creations